### DISCUSSION TITLE
When closing a tab, open the previous one

### DISCUSSION THREAD

**Discussion by AlessandroPet05** [primefaces/community#1247] [PrimeNG] [answered]
Hi everyone, I would like to know if there was a way to ensure that when I close a tab of the TabView component the previous one opens. I notice that PrimeNG only allows the main tab to be opened anyway.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

You can do this by utilizing `activeIndex` property and `onClose` event of tabView. Here is a basic stackblitz example https://stackblitz.com/run?file=src%2Fapp%2Fdemo%2Ftab-view-scrollable-demo.ts,src%2Fapp%2Fdemo%2Ftab-view-scrollable-demo.html

**Reply by AlessandroPet05** [reply]
How can I open this link? It simply takes me back to the main stackblitz page
